benchmarks/benchmark_experiment.py

```diff
@@ -138,6 +150,9 @@ def update_process_env(self, process_env):
       process_env.pop("XRT_TPU_CONFIG", None)
       process_env.pop("XLA_FLAGS", None)
 
+    if self.use_torch_xla2:
+      process_env.pop("JAX_PLATFORMS", self.accelerator.upper())
```

Comment:
Could you add a comment why we need to remove this env var?

Reply:
Hah, typo here. Insert the env instead of pop. Thanks